bin/generate-gutenberg-php.php

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Generates the production (plugin build) version of `gutenberg.php`,
+ * containing alternate `define` statements from the development version.
+ *
+ * @package gutenberg-build
+ */
+
+$f = fopen( dirname( dirname( __FILE__ ) ) . '/gutenberg.php', 'r' );
+
+$plugin_version = null;
+$inside_defines = false;
+
+/**
+ * Prints `define` statements for the production version of `gutenberg.php`
+ * (the plugin entry point).
+ */
+function print_production_defines() {
+	global $plugin_version;
+
+	echo "define( 'GUTENBERG_VERSION', '$plugin_version' );\n";
```

Comment:
Can you explain why we want these defines? They don't appear to be used? Is it just because we need to replace the development mode constant so we might as well? Wondering if it'd help simplify this at all if we just moved to stripping out the define section during packaging.